src/shared/components/paginationControls/paginationControls.module.scss

```diff
@@ -14,4 +14,8 @@
     height: 18px;
     padding: 0 5px 0 5px;
     display: inline-block;
+}
+
+:global(.legacy) .paginationControls button i {
```

Comment:
using legacy here so that it only applies when on pages with html[class=legacy].  When we are finished with refactor, we can then delete legacy